### ISSUE TITLE
Predicate function names check does not work for non-zero arity functions

### ISSUE THREAD

**Issue by laurglia** [rrrene/credo#866]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 1.5.4
* Erlang/Elixir version (`elixir -v`): Erlang 23, Elixir 1.11.3
* Operating system: Mac OS

### What were you trying to do?

1. Have following function definition

```elixir
defmodule A do
  defp is_user(cookie) do
    true
  end
end
```

2. Credo has `PredicateFunctionNames` check enabled.
3. Run credo

### Expected outcome

Credo reports an issue with the function - it is a predicate function that starts with `is_`.

### Actual outcome

Credo does not report any issues.

### Extra information

The check works fine if I make the function 0-arity, for the following code an issue is reported:

```elixir
defmodule A do
  defp is_user do
    true
  end
end
```

**Comment by panoramix360** [comment]
I created a pull request fixing this [issue](https://github.com/rrrene/credo/pull/867), just waiting for approval :)

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by rrrene** [comment]
This is part of v1.6.1 💚
